src/stores/Tabs/tab.js

```diff
@@ -16,6 +16,12 @@ import { TabSelectedItems } from "./tab_selected_items";
 import { History } from '../../utils/history';
 import { FF_LOPS_12, isFF } from "../../utils/feature-flags";
 
+const ThresholdModel = types
+  .model("ThresholdModel", {
+    from: types.optional(types.number, 0),
+    to: types.optional(types.number, 100),
+  });
+
```

Comment:
Is there any reason not to move this out to its own model? I don't see much relation to Tabs here, all the data is separate and should be separate to Datamanager.

Reply:
no reason not to move this into a separate model file - we just dont use it else where - will do